cyclopts/help/help.py

```diff
@@ -225,84 +359,59 @@ def __rich_console__(self, console: "Console", options: "ConsoleOptions") -> "Re
         if not self.entries:
             return _silent
 
-        import textwrap
-
-        from rich.box import ROUNDED
         from rich.console import Group as RichGroup
         from rich.console import NewLine
-        from rich.panel import Panel
-        from rich.table import Table
         from rich.text import Text
 
-        wrap = partial(
-            textwrap.wrap,
-            subsequent_indent="  ",
-            break_on_hyphens=False,
-            tabsize=4,
-        )
-        # (top, right, bottom, left)
-        table = Table.grid(padding=(0, 2, 0, 0), pad_edge=False)
-        panel_description = self.description
+        commands_width = ceil(console.width * 0.35)
```

Comment:
these width calculations no longer belong here, right? I think this belongs in `ColumnSpec`?